src/pages/Jobs/AllJobs/index.jsx

```diff
@@ -16,20 +17,179 @@
+        {showMessage && (
+          <Alert
+            type={message.type}
+            setShow={setShowMessage}
+            message={message.text}
+          />
+        )}
         <div className="row">
-          <div className="col-lg-8 col-md-12 col-sm-12 col-12">
-            <h1 className="font-size-main-heading">Show Jobs</h1>
-            <br />
+          <div className="col-md-3 col-lg-2">
+            <h2 className="font-size-sub-heading">All Jobs</h2>
+          </div>
+          <div className="col-md-9 col-lg-10">
+            <table className="table table-striped text-primary-color font-size-medium table-responsive-sm table-bordered">
+              <thead>
+                <tr>
+                  <th colSpan="6" className="font-size-sub-heading text-center">
+                    All Recent Jobs
+                  </th>
+                </tr>
+                <tr className="font-demi">
+                  <td colSpan="6" className="p-2 m-2 text-left">
+                    <span className="mt-1 ">Show entries: </span>
+                    <InputContainer
+                      name="limit"
+                      type="select"
+                      onChange={(e) => handleChange(e)}
+                      options={entriesOptions}
+                      property="entry"
+                    />
+                  </td>
+                </tr>
+                <tr>
+                  <th>
+                    <input
+                      type="search"
+                      className="form-control"
+                      placeholder="Filter Jobs"
+                      onChange={(event) => setQuery(event.target.value)}
+                    />
+                  </th>
+                  <th />
+                </tr>
+                <tr className="font-bold text-center">
+                  <th>Job Name</th>
+                  <th>Queue Date</th>
+                  <th>Status</th>
+                </tr>
+              </thead>
+              <tbody>
+                {jobsDataList
+                  ?.filter((post) => {
+                    if (query === "") {
+                      return post;
+                    }
+                    if (post?.name.toLowerCase().includes(query.toLowerCase()))
+                      return post;
+                    return null;
+                  })
+                  .map((data) => (
+                    <tr className="text-center">
+                      <td>
+                        <div className="font-demi">{data?.name}</div>
+                      </td>
+                      <td>
+                        <div className="font-demi">{data?.queueDate}</div>
+                      </td>
+                      <td>{data?.status}</td>
+                    </tr>
+                  ))}
+                <tr className="text-left">
+                  <td colSpan="6">
+                    Page:
+                    {pagesOptions && (
+                      <InputContainer
```

Comment:
Can pagination like this be used **iff it is not a big effort**?
![image](https://user-images.githubusercontent.com/18077542/175230696-baa4d745-1c53-4871-bc2e-437f18e4e22b.png)

Reply:
Yes we can implement this but I think this particular PR has been a very lengthy one. I plan to introduce this feature once this PR is closed. Cause once this one is closed at least the page will be introduced in the UI in a functioning state.

Reply:
Sure. Please create an issue for it once the PR merges.